admin/templates/registration/password_change_done.html

```diff
@@ -0,0 +1,17 @@
+{% extends "admin/base_site.html" %}
+{% load i18n %}
+{% block userlinks %}{% url 'django-admindocs-docroot' as docsroot %}{% if docsroot %}<a href="{{ docsroot }}">{% trans 'Documentation' %}</a> / {% endif %}{% trans 'Change password' %} / <a href="{% url 'admin:logout' %}">{% trans 'Log out' %}</a>{% endblock %}
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; {% trans 'Password change' %}
+</div>
+{% endblock %}
+
+{% block title %}{{ title }}{% endblock %}
+{% block content_title %}<h1>{{ title }}</h1>{% endblock %}
+{% block content %}
+<p>{% trans 'Your password was changed.' %}</p>
```

Comment:
Getting setup for translating is a good idea but there are a lot of places throughout this PR where we aren't doing it. Not sure what the requirement for i18n was for the story but my vote would be do it everywhere we have strings or don't do it now. It'll be a lot easier to add it in later if it's one or the other.